public/global.css

```diff
@@ -54,10 +63,10 @@ pre, code, samp {
  */
 .button {
   display: inline-block;
-  margin: 0 .625em;
-  padding: .625rem 1.25rem; /* button--medium size */
+  margin: 0 0.625em;
+  padding: 0.625rem 1.25rem; /* button--medium size */
   border: 1px solid transparent;
-  border-radius: .625rem;
+  border-radius: 0.625rem;
```

Comment:
Did your editor's linter do this?

Reply:
lol yeah my Prettier plug-in added all of these 0. Should I remove these zeroes

Reply:
Yes please! Remove all the zeros haha!